src/core/EnvelopeAndLfoParameters.cpp

```diff
@@ -89,33 +86,36 @@ void EnvelopeAndLfoParameters::LfoInstances::remove( EnvelopeAndLfoParameters *
 
 
 EnvelopeAndLfoParameters::EnvelopeAndLfoParameters(
-					float _value_for_zero_amount,
-							Model * _parent ) :
-	Model( _parent ),
+    float _value_for_zero_amount,
+    Model * _parent ) :    
+    Model( _parent ),
 	m_used( false ),
+    
+ 
+//  m_predelayModel( val, min, max, incr,  this, tr( ">info<" ) )   
 	m_predelayModel( 0.0, 0.0, 2.0, 0.001, this, tr( "Env pre-delay" ) ),
-	m_attackModel( 0.0, 0.0, 2.0, 0.001, this, tr( "Env attack" ) ),
-	m_holdModel( 0.5, 0.0, 2.0, 0.001, this, tr( "Env hold" ) ),
-	m_decayModel( 0.5, 0.0, 2.0, 0.001, this, tr( "Env decay" ) ),
-	m_sustainModel( 0.5, 0.0, 1.0, 0.001, this, tr( "Env sustain" ) ),
-	m_releaseModel( 0.1, 0.0, 2.0, 0.001, this, tr( "Env release" ) ),
-	m_amountModel( 0.0, -1.0, 1.0, 0.005, this, tr( "Env mod amount" ) ),
+	m_attackModel( 0.025, 0.0, 2.0, 0.001, this, tr( "Env attack" ) ),
+	m_holdModel( 0.0, 0.0, 2.0, 0.001,     this, tr( "Env hold" ) ),
+	m_decayModel( 0.6, 0.0, 2.0, 0.001,    this, tr( "Env decay" ) ),
+	m_sustainModel( 0.0, 0.0, 1.0, 0.001,  this, tr( "Env sustain" ) ),
+	m_releaseModel( 0.2, 0.0, 2.0, 0.001,  this, tr( "Env release" ) ),
+	m_amountModel( 1.0, -1.0, 1.0, 0.005,  this, tr( "Env mod amount" ) ), //AMP 
 	m_valueForZeroAmount( _value_for_zero_amount ),
 	m_pahdFrames( 0 ),
 	m_rFrames( 0 ),
 	m_pahdEnv( NULL ),
 	m_rEnv( NULL ),
 	m_pahdBufSize( 0 ),
 	m_rBufSize( 0 ),
-	m_lfoPredelayModel( 0.0, 0.0, 1.0, 0.001, this, tr( "LFO pre-delay" ) ),
-	m_lfoAttackModel( 0.0, 0.0, 1.0, 0.001, this, tr( "LFO attack" ) ),
+	m_lfoPredelayModel( 0.0, 0.0, 1.0, 0.001, this, tr( "LFO Predelay" ) ),
+	m_lfoAttackModel( 0.0, 0.0, 1.0, 0.001, this, tr( "LFO Attack" ) ),
 	m_lfoSpeedModel( 0.1, 0.001, 1.0, 0.0001,
-				SECS_PER_LFO_OSCILLATION * 1000.0, this,
-							tr( "LFO frequency" ) ),
-	m_lfoAmountModel( 0.0, -1.0, 1.0, 0.005, this, tr( "LFO mod amount" ) ),
-	m_lfoWaveModel( SineWave, 0, NumLfoShapes, this, tr( "LFO wave shape" ) ),
-	m_x100Model( false, this, tr( "LFO frequency x 100" ) ),
-	m_controlEnvAmountModel( false, this, tr( "Modulate env amount" ) ),
+	SECS_PER_LFO_OSCILLATION * 1000.0, this,
+	tr( "LFO speed" ) ),
+	m_lfoAmountModel( 0.0, -1.0, 1.0, 0.005, this, tr( "LFO Modulation" ) ),
+	m_lfoWaveModel( SineWave, 0, NumLfoShapes, this, tr( "LFO Wave Shape" ) ),
+	m_x100Model( false, this, tr( "Freq x 100" ) ),
+	m_controlEnvAmountModel( false, this, tr( "Modulate Env-Amount" ) ),
```

Comment:
Will updating these break translations?

I'm also not sure about the changes here - I like `frequency` -> `speed`, but I'm not as sure about `env amount` -> `Env-Amount`. This makes the capitalization inconsistent, as it hasn't been changed above (above is first word has first letter capitalized: 'Env attack', 'Env release', etc, while this is all words have first letter capitalized: 'LFO Modulation', 'LFO Wave Shape', etc).

Also, `Env mod amount` has not been changed, so I think changing `LFO mod amount` to `LFO Modulation` could be confusing to a new user - they both do the same thing for their respective modulators, i.e. control mod amount, so naming them in a similar way makes the most sense to me.

I would personally suggest leaving the names unchanged for this PR and submitting a separate PR for name changes that also maintains consistency across LMMS, and also that deals with any translation issues.